requirements.txt

```diff
@@ -3,9 +3,9 @@ psutil
 ray >= 2.9
 sentencepiece  # Required for LLaMA tokenizer.
 numpy
-torch == 2.1.2
+torch == 2.2.0
 transformers >= 4.38.0  # Required for Gemma.
-xformers == 0.0.23.post1  # Required for CUDA 12.1.
+xformers == 0.0.24  # Required for CUDA 12.1.
```

Comment:
```suggestion
xformers == 0.0.24  # Required for PyTorch 2.2.0.
```

Reply:
@hmellor Could you make this change? I think we can merge the PR after fixing this and fixing a small issue in CI.